asreview/webapp/src/PreReviewComponents/ProjectInit.js

```diff
@@ -0,0 +1,237 @@
+import React from "react";
```

Comment:
@terrymyc was this file moved on master?

Reply:
This file was renamed as `ProjectInfo.js`.